### ISSUE TITLE
Admin plugin to increase the validity of the membership

### ISSUE THREAD

**Issue by all85100** [alliedmodders/amxmodx#572]
Admin plugin to increase the validity of the membership, in Users.ini set the validity period so that the member management more convenient.
I have someone else modified plug-ins, can learn from the next, so that can be modified faster! Plugins: Dl.lonzi.cn/cstrike/admin.sma, configuration files: Http://dl.lonzi.cn/cstrike/users.ini

**Comment by Arkshine** [comment]
Can you do your changes on the `admin.sma` from 1.9 packages and posting a link again? Looks like you are using a very old version. It's really hard to see what you did and you are proposing.

**Comment by all85100** [comment]
No, I mean in the admin plugin add a limit membership validity, my old plugin has this function is someone else to change, I mean you can see for reference, you can quickly increase this function, of course, you can not learn

不是,我是说在admin插件上增加一个限制会员有效期的功能,我的这个老插件有这个功能是别人改的,我意思是你看看可以借鉴一下,可以很快的增加这个功能,当然也可以不用借鉴

**Comment by Arkshine** [comment]
Oh, you mean temporary admins. Like, for example: https://forums.alliedmods.net/showthread.php?p=112793 ?

**Comment by HamletEagle** [comment]
Yes, he wants the option to set an expiration date for admins.

**Comment by all85100** [comment]
https://forums.alliedmods.net/showthread.php?p=112793? 
This does not seem to me this good, but also added the configuration file, if you read my this plugin you will understand that this is very good, hoping to increase

**Comment by OciXCrom** [comment]
It should be pretty easy to implement. An optional param needs to be added to users.ini which will hold the date and when reading the file, simply skip the line if the date has expired. Maybe also a cvar if the admin should automatically get removed from the file. I may give it a try later.

**Comment by all85100** [comment]
OK,thinks
